src/material-experimental/mdc-select/select.ts

```diff
@@ -112,6 +112,8 @@ export class MatSelect extends _MatSelectBase<MatSelectChange> implements OnInit
   /** Width of the overlay panel. */
   _overlayWidth: number;
 
+  _overlayPanelClass: string | string[] = this._defaultOptions?.overlayPanelClass || '';
```

Comment:
You can put this on the base class instead of repeating it on the two sub-classes.